src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -15,13 +15,7 @@
  */
 package com.squareup.kotlinpoet
 
-import com.squareup.kotlinpoet.KModifier.ABSTRACT
-import com.squareup.kotlinpoet.KModifier.ANNOTATION
-import com.squareup.kotlinpoet.KModifier.COMPANION
-import com.squareup.kotlinpoet.KModifier.ENUM
-import com.squareup.kotlinpoet.KModifier.EXPECT
-import com.squareup.kotlinpoet.KModifier.EXTERNAL
-import com.squareup.kotlinpoet.KModifier.PUBLIC
+import com.squareup.kotlinpoet.KModifier.*
```

Comment:
We usually don't use wildcard imports, you can configure your IDE to not squash imports